survey-sdk/src/main/java/co/nimblehq/survey/sdk/SurveyApi.kt

```diff
@@ -23,6 +24,15 @@ class SurveyApi private constructor() : NetworkBuilder() {
         return this
     }
 
+    fun withCredentials(clientID: String, clientSecret: String) {
+        BaseRequest.updateAuthenticationInfo(clientID, clientSecret)
+    }
+
+    fun setTokenApi(token: String): SurveyApi {
```

Comment:
OPTIONAL: `setToken`

Reply:
@hoangnguyen92dn I prefer to change `setTokenKey` to avoid overwrite its parent.